nncf/torch/graph/operator_metatypes.py

```diff
@@ -1101,6 +1101,24 @@ class PTScaledDotProductAttentionMetatype(PTOperatorMetatype):
     target_input_ports = [0, 1]
 
 
+@PT_OPERATOR_METATYPES.register()
+class PTEmptyMetatype(PTOperatorMetatype):
+    name = "EmptyOp"
+    module_to_function_names = {NamespaceTarget.TORCH: ["empty"]}
+
+
+@PT_OPERATOR_METATYPES.register()
+class PTVectorNormMetatype(PTOperatorMetatype):
+    name = "VectorNormOp"
+    module_to_function_names = {NamespaceTarget.ATEN: ["linalg_vector_norm"]}
+
+
+@PT_OPERATOR_METATYPES.register()
+class PTClampMetatype(PTOperatorMetatype):
+    name = "ClampOp"
+    module_to_function_names = {NamespaceTarget.TORCH: ["clamp", "clamp_min"]}
+
+
```

Comment:
As this metatypes don't have any specific affect on any algorithm, I suggest to remove them. In that case following operation will have `UnknownMetatype` and will be treated the same way as now.

Reply:
That sounds good, Alright.

Reply:
Done!

Reply:
Don't forget to update references as well

Reply:
Done!